rt/mlc/OPTB_z12.OPT

```diff
@@ -0,0 +1,6 @@
+noallow
+  machine(Z12,list)
+  stats
+noloadhigh
+nodump
+notest
```

Comment:
I assume the lowercase 'z" in the file name is correct. Same comment for next two files.

Reply:
yes, that was done on purpose